tests/Unit/Parallel/Test_ResourceInfo.cpp

```diff
@@ -271,62 +208,31 @@ void test_single_core() {
     CHECK(info_0.is_exclusive() == serialized_info_0.is_exclusive());
   }
   {
-    INFO("Only Singletons");
-    using metavars = MetavariablesInfoOnly<0, 1>;
-    Parallel::MutableGlobalCache<metavars> mutable_cache{};
-    Parallel::GlobalCache<metavars> cache{{}, &mutable_cache};
-    // Both of these should be identical since we are running on one proc
+    INFO("Singletons::Auto");
+    using metavars = Metavariables<0, 1>;
     auto resource_info =
         TestHelpers::test_option_tag<OptionTags::ResourceInfo<metavars>>(
-            "Singletons:\n"
-            "  FakeSingleton0:\n"
-            "    Proc: 0\n"
-            "    Exclusive: false\n"
-            "  FakeSingleton1:\n"
-            "    Proc: Auto\n"
-            "    Exclusive: false\n");
-
+            "AvoidGlobalProc0: false\n"
+            "Singletons: Auto\n");
+    Parallel::MutableGlobalCache<metavars> mutable_cache{};
+    Parallel::GlobalCache<metavars> cache{{}, &mutable_cache};
     resource_info.build_singleton_map(cache);
 
-    const size_t proc_0 =
-        resource_info.template proc_for<FakeSingletonInfoOnly<metavars, 0>>();
-    const size_t proc_1 =
-        resource_info.template proc_for<FakeSingletonInfoOnly<metavars, 1>>();
-
-    // Only running on once proc
-    CHECK(proc_0 == 0);
-    CHECK(proc_1 == 0);
```

Comment:
Could you keep these checks in? I know the procs should be the same as the ones below that were gotten from the singleton info's, but that's what we're testing. That they are the same, just gotten different ways.